core/templates/dev/head/components/share/sharing_links_directive.html

```diff
@@ -7,13 +7,13 @@
 
   <li>
     <a ng-href="https://www.facebook.com/sharer/sharer.php?sdk=joey&u=<[serverName]>/<[activityType]>/<[activityId]>&display=popup&ref=plugin&src=share_button" onclick="return !window.open(this.href, '', 'height=336, width=640')" ng-click="registerShareEvent('facebook')" target="_window">
-      <img class="share-option-img" ng-src="<[fbUrl]>" alt="Facebook">
+      <img class="share-option-img" ng-src="fa fa-facebook" alt="Facebook">
```

Comment:
@vibhor98 , Why is this changed and also the twitter image src ?

Reply:
I must have changed this file earlier to solve other issue. But by mistake, I have added this   
In this PR. Sorry for that. Can you remove this edited  file? Thanks!